src/MSALWrapper/AuthFlow/IntegratedWindowsAuthentication.cs

```diff
@@ -109,6 +109,10 @@ public async Task<AuthFlowResult> GetTokenAsync()
             {
                 this.logger.LogWarning($"Msal Client Exception! (Not expected)\n{ex.Message}");
                 this.errors.Add(ex);
+                if (ex.Message.StartsWith("WS-Trust endpoint not found", StringComparison.OrdinalIgnoreCase))

```

Comment:
Does the message really start with this or is there other content in there? Have you debugged this to see the actual message while your VPN is off?

Reply:
Here's the terminal o/p when VPN is off
```
PS C:\> azureauth --mode=iwa --alias ado --config c:\data\azureauth.toml
Msal Client Exception! (Not expected)
WS-Trust endpoint not found in metadata document.
```

Reply:
Since we are expecting the exact message, I think we don't need to ignore case. :)

Reply:
Yes, Checked with MSALs error message. But to be on the safe side, if MSAL were to update it with a prefix, I am updating this to check if it contains the same content.